pkg/storage/etcd3/compact.go

```diff
@@ -25,7 +26,10 @@ import (
 	"golang.org/x/net/context"
 )
 
-const compactInterval = 10 * time.Minute
+const (
+	compactInterval = 10 * time.Minute
```

Comment:
> What's the reasoning behind the 10 minute interval?

There is already comment in there:
"We save the most recent 10 minutes of data which should be enough for slow watcher and to tolerate burst. Ideally, when the storage layer can take advantage of mvcc, we might increase the interval to 12h."

> how would compaction affect etcd performance when under pressure?

It will slight affect, but won't pause.